services/pepy/pepy-downloads.service.js

```diff
@@ -0,0 +1,53 @@
+import Joi from 'joi'
+import { nonNegativeInteger } from '../validators.js'
+import { BaseJsonService, pathParams } from '../index.js'
+import { renderDownloadsBadge } from '../downloads.js'
+
+const schema = Joi.object({
+  total_downloads: nonNegativeInteger,
+}).required()
+
+const description = `
+Python package total downloads from [Pepy](https://www.pepy.tech/).
+
+Download stats from pepy count package downloads from PyPI and known mirrors.`
+
+export default class PepyDownloads extends BaseJsonService {
+  static category = 'downloads'
+
+  static route = {
+    base: 'pepy',
+    pattern: 'dt/:packageName',
```

Comment:
I've only added total for now, but in principle we could make this `:period(dw|dm|dt)` or something in future without introducing a redirect